src/Types.ts

```diff
@@ -776,6 +785,23 @@ export type RootConstructor = (
     props: RootProps
 ) => Root;
 
+/**
+ * Ability to fine-tune Tabster internal behaviour in rare cases of need.
+ * Normally, should not be used. A deep understanding of the intention and the effect
+ * is required.
+ */
+export interface SysProps {
+    /**
+     * Force dummy input position outside or inside of the element.
+     * By default (when undefined), the position is determined dynamically
+     * (for example inside for <li> elements and outside for <table> elements,
+     * plus a default Groupper/Mover/Modalizer implementation position).
+     * Setting to true will force the dummy inputs to be always outside of the element,
+     * setting to false will force the dummy inputs to be always inside.
+     */
+    dummyInputsOutside?: boolean;
```

Comment:
This should be an enum, having Boolean properties that do something different on undefined kind of smells

Reply:
I've made it a Tabster-style enum (which I hate but it's much better from the final bundle perspective comparing to what TypeScirpt does to enums).